frontend/src/lang/modules/en.ts

```diff
@@ -1246,7 +1246,7 @@ const message = {
         infoDetail: 'File properties',
         root: 'Root directory',
         list: 'File list',
-        sub: 'Include subdirectories',
+        sub: 'Subdirectories',
         downloadSuccess: 'Successfully downloaded',
         theme: 'Theme',
         language: 'Language',
```

Comment:
The difference between `sub` values is that `include subdirectories` has been updated to simply `Subdirectories`. This should not cause any major issues but could be confusing for users who rely on the original wording.